rerun_py/rerun_sdk/rerun/_spawn.py

```diff
@@ -55,11 +55,6 @@ def _spawn_viewer(
         return
     new_env["RERUN_APP_ONLY"] = "true"
 
-    # sys.executable: the absolute path of the executable binary for the Python interpreter
-    python_executable = sys.executable
-    if python_executable is None:
-        python_executable = "python3"
-
     # TODO(jleibs): More options to opt out of this behavior.
     if _check_for_existing_viewer(port):
```

Comment:
This could be removed if we used `RecordingStream::spawn_opts`

Reply:
We are calling the same function that spawn_ops calls ans passing through the SpawnOpts struct. Not using the spawn_ops on Recording stream because of some details around how we manage sinks and blueprints though. Good call that this stuff is redundant with the implementation inside of spawn though.